fail to vagrant up after I download the "openstreetmap-website"  file  according to the step

I use vagrant and VirtualBox Built a virtual environment on my Mac，and download the "openstreetmap-website"  file  according to the step. when i  vagrant up，default: make: *** No rule to make target '`/root`', needed by 'quadtile.o'.  Stop.   How do i fix it? Thanks.

--- a/db/functions/Makefile
+++ b/db/functions/Makefile
@@ -1,7 +1,7 @@
 PG_CONFIG ?= pg_config
 DESTDIR ?= .
 
-QTDIR=$(shell bundle show quad_tile)/ext/quad_tile
+QTDIR=$(shell bundle show quad_tile | tail -n 1)/ext/quad_tile
 
 OS=$(shell uname -s)
 ifeq (${OS},Darwin)

